docs/scu/dev/onboard-to-spryker-code-upgrader/prepare-a-project-for-spryker-code-upgrader.md

```diff
@@ -30,6 +24,10 @@ Currently, the Upgrader supports GitHub and GitLab. If you want to use a differe
 
 The Upgrader does not evaluate frontend customizations. You can either move to headless or apply frontend upgrades manually.
 
+## Optional: Make your code is compliant with supported extensions scenarios
```

Comment:
I believe it shouldn't be optional

Reply:
As I understood, Vova doesn't want to obligate customers, to prepare compatibility of their projects with all our rules. the current idea is: "You can use Upgrader right now, but if your project is not compatible with some rules, you will not receive some updates"

Reply:
ok